src/libutil/strlcpy.h

```diff
@@ -3,6 +3,8 @@
 #endif /* HAVE_CONFIG_H */
 
 #if !HAVE_STRLCPY
+#define HAVE_STRLCPY
+
```

Comment:
That name reminds me of an autoconf test result?    Might be good to use a name more like the other header guards, like
```
#ifndef _UTIL_STRLCPY_H
#define _UTIL_STRLCPY_H
...
#endif // !_UTIL_STRLCPY_H
```